common/tests/integration_tests.rs

```diff
@@ -128,12 +129,201 @@ fn basic2() {
         .collect::<Vec<_>>();
     csv.verify_last_header_finalization(&fp).unwrap();
     light_client.update(block_header, fp).unwrap();
+    let commits = csv.get_total_commits();
     let merkle_tree = OneshotMerkleTree::create(
-        csv.get_total_commits()[1..=3]
+        commits[1..=(commits.len() - 2)]
             .iter()
             .map(|c| c.to_hash256())
             .collect(),
     );
     let merkle_proof = merkle_tree.create_merkle_proof(tx.to_hash256()).unwrap();
     assert!(light_client.verify_transaction_commitment(&tx, 1, merkle_proof));
 }
+
+#[test]
+fn basic3() {
+    let member_number = 10;
+    let (rs, keys) = test_utils::generate_standard_genesis(member_number);
+    let genesis_info = rs.genesis_info.clone();
+    let genesis_header = rs.genesis_info.header.clone();
+
+    let (height, timestamp) = (1, 0);
```

Comment:
I think it's better to keep the same manner within tests. Let's just hard code `height` and `timestamp`.